src/libmongoc/src/mongoc/mongoc-topology-description.c

```diff
@@ -655,15 +655,17 @@ _mongoc_topology_description_validate_max_staleness (
 }
 
 static bool
-_check_any_server_less_than_wire_version_v5_0 (const void *sd_,
-                                               void *any_too_old_)
+_check_any_server_less_than_wire_version_13 (const void *sd_,
+                                             void *any_too_old_)
 {
    const mongoc_server_description_t *sd = sd_;
    bool *any_too_old = any_too_old_;
-   if (sd->max_wire_version < WIRE_VERSION_5_0) {
+   if (sd->max_wire_version < 13) {
```

Comment:
```suggestion
   if (sd->max_wire_version < WIRE_VERSION_5_0) {
```
Reverted; should still be applied.